licenses/MADlib.txt

```diff
@@ -1,10 +0,0 @@
-Portions of this software Copyright (c) 2010-2013 by EMC Corporation.  All rights reserved.
```

Comment:
This file is actually being replaced with a symlink to ../LICENSE -- it seems to be necessary for Mac OS X packaging. I can also replace it with a copy of LICENSE, but that'll be a tad messy.